test/test_helper.rb

```diff
@@ -7,5 +7,5 @@
 require "minitest/autorun"
 
 $LOAD_PATH.unshift(File.join(File.expand_path(__dir__), "..", "lib"))
-require "safe.rb"
+require "safe_eth_ruby.rb"
```

Comment:
The Ruby interpreter will automatically look for a .rb file when requiring dependencies.